javalin/src/main/java/io/javalin/jetty/JavalinJettyServlet.kt

```diff
@@ -70,8 +72,12 @@ class JavalinJettyServlet(val cfg: JavalinConfig, private val httpServlet: Javal
         super.service(req, res) // everything is okay, perform websocket upgrade
     }
 
+    private val accessManagerHandler = Handler { it.attribute("javalin-ws-upgrade-allowed", true)}
+
     private fun allowedByAccessManager(entry: WsEntry, ctx: Context): Boolean = try {
-        cfg.pvt.accessManager.manage({ it.attribute("javalin-ws-upgrade-allowed", true) }, ctx, entry.roles) // run access manager
+        cfg.pvt.accessManager
+            ?.manage(accessManagerHandler, ctx, entry.roles) // run custom access manager
+            ?: SecurityUtil.noopAccessManager(accessManagerHandler, ctx, entry.roles)
```

Comment:
Can this be de-duplicated?

Reply:
Not really, because logic inside those two is slightly different, so utility function should be quite abstract and in the end it won;t help that much imo